### DISCUSSION TITLE
DST related panic

### DISCUSSION THREAD

**Discussion by imran-iq** [kdheepak/taskwarrior-tui#541] [General]
Just starting a discussion here since it is not a bug in taskwarrior-tui itself but in taskwarrior: https://github.com/GothenburgBitFactory/taskwarrior/issues/75

I assume since it around the DST switch over that long living repeated tasks end up eventually having invalid dates (which causes the application to panic with:
```
The application panicked (crashed).
  No such local time
in src/task_report.rs, line 22
thread: main
```

Run the script linked in the issue fixed the issue for me

**Comment by kdheepak** [comment]
Wow, I had no idea. Thanks for reporting! I'm going to make it an issue, since taskwarrior-tui should better report this error at the very least.
